pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick an option to print to `STDOUT`:
+
+`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}"`
+
+- Open an interactive prompt for the user to input a string with a placeholder:
+
+`gum input --placeholder "{{placeholder_string}}"`
+
+- Interactvely pick an option which will exit if "No" is chosen:
+
+`gum confirm "{{Continue?}}" --default=false --affirmative "{{Yes}}" --negative "{{No}}" {{&& : || exit 1}}`
+
+- Show a spinner while a command is taking place with text alongside:
+
+`gum spin --spinner {{dot|line|minidot|jump|pulse|points|globe|moon|monkey|meter|hamburger}} --title "{{loading...}}" -- {{command}}`
```

Comment:
I don't like too many options listed inside curly braces. I think we should not do that not to introduce extra complexity in tldr pages. Maybe we can limit alternatives up to 5 (inclusive).

Reply:
Yeah, I too think the same. Maybe we can reduce it to some essential examples as displaying it would be complex for the client. 
(Or)
We can provide only one example as given in the [Github Page](https://github.com/charmbracelet/gum#spin).
```suggestion
`gum spin --spinner {{dot}} --title "{{title}}" -- {{command}}`
```

Reply:
see `tldr nmcli`, its better to just show all of the options since otherwise the user may not know that the option exists. The spinner choices aren't well documented from what I can see, I found them by going [here](https://github.com/charmbracelet/gum/blob/main/spin/spinners.go)

Reply:
I think we can list all options just when docs don't describe them all and user have to dig into source code to understand what can be used.